proposals/0540-jsem.rst

```diff
@@ -171,18 +175,26 @@ know anything about the jobserver that has spawned them.
 Implementation of the protocol
```

Comment:
This is more than a mere implementation matter.   The `semaphore-compat` library is part of the specification.   Maybe re-title this subsection?